src/translators/sql_translator.py

```diff
@@ -1107,13 +1108,18 @@ def query(self,
                  )
             try:
                 self.cursor.execute(op)
+
             except Exception as e:
                 # TODO due to this except in case of sql errors,
                 # all goes fine, and users gets 404 as result
                 # Reason 1: fiware_service_path column in legacy dbs.
-                self.sql_error_handler(e)
+                err_msg = self.sql_error_handler(e)
                 self.logger.error(str(e), exc_info=True)
                 entities = []
+                if err_msg:
+                    return (result, err_msg)
```

Comment:
in the end what we need to change is the message, not the 'result' (that was handled correctly also before)

so, i propose:
- before exception set message = 'ok'
- in the exception set the message as needed (but don't return anything, just set the message value)
- in the final return use do `return (result, message)`

Reply:
done